packages/framework/resources/views/layouts/publication.blade.php

```diff
@@ -0,0 +1,9 @@
+{{-- The Publication Page Layout --}}
+@extends('hyde::layouts.app')
+@section('content')
+
+    <main id="content" class="mx-auto max-w-7xl py-16 px-8">
+        {{ $publication->title }}
+    </main>
+
+@endsection
```

Comment:
Is this file used?

Reply:
Haven't checked yet, but we could merge these files to use the stub contents in the type create action

Reply:
Resolved in https://github.com/hydephp/develop/pull/807